homeassistant/components/motionblinds_ble/const.py

```diff
@@ -0,0 +1,64 @@
+"""Constants for the MotionBlinds BLE integration."""
+
+from enum import StrEnum
+
+ATTR_CALIBRATION = "calibration"
+ATTR_CONNECTION_TYPE = "connection_type"
+ATTR_SPEED = "speed"
+ATTR_BATTERY = "battery"
+ATTR_CONNECT = "connect"
+ATTR_DISCONNECT = "disconnect"
+ATTR_FAVORITE = "favorite"
+ATTR_SIGNAL_STRENGTH = "signal_strength"
+
+CONF_LOCAL_NAME = "local_name"
+CONF_ADDRESS = "address"
+CONF_MAC_CODE = "mac_code"
+CONF_BLIND_TYPE = "blind_type"
+
+DOMAIN = "motionblinds_ble"
+
+ENTITY_NAME = "MotionBlind {mac_code}"
+
+ERROR_ALREADY_CONFIGURED = "already_configured"
+ERROR_COULD_NOT_FIND_MOTOR = "could_not_find_motor"
+ERROR_INVALID_MAC_CODE = "invalid_mac_code"
+ERROR_NO_BLUETOOTH_ADAPTER = "no_bluetooth_adapter"
+ERROR_NO_DEVICES_FOUND = "no_devices_found"
+
+EXCEPTION_NOT_CALIBRATED = (
+    "{device_name} needs to be calibrated using the MotionBlinds BLE app before usage."
+)
+
+ICON_CALIBRATION = "mdi:tune"
+ICON_SPEED = "mdi:run-fast"
+ICON_CONNECT = "mdi:bluetooth"
+ICON_DISCONNECT = "mdi:bluetooth-off"
+ICON_FAVORITE = "mdi:star"
+ICON_CONNECTION_TYPE = "mdi:bluetooth-connect"
+ICON_VERTICAL_BLIND = "mdi:blinds-vertical-closed"
```

Comment:
These icons should be defined in `icons.json`.

Reply:
Will add them there thanks!

Reply:
Current the current cover platform I only really need `ICON_VERTICAL_BLIND`, which I use here:

Reply:
In this case I only need the `ICON_VERTICAL_BLIND` as the others are related to the other platforms. Currently I use the translation_key for the translation of a blind attribute, which is the same for all types of blinds. If I want to use icons.json, I have to use a different translation_key for the vertical blind right, but then it doesn't use the attribute translation anymore since I changed the translation_key from `blind` to `vertical`? How can I solve this without defining the translation twice exactly the same for `blind` and `vertical`?